test/libsolidity/syntaxTests/nameAndTypeResolution/276_invalid_types_in_inline_array.sol

```diff
@@ -1,8 +1,7 @@
 contract C {
     function f() public {
-        uint[3] x = [45, 'foo', true];
```

Comment:
I think this was rather meant to be `memory`.

Reply:
Yes, done.